releasenotes/notes/pulse_parameter_manager_compat_with_ParameterVector-7d31395fd4019827.yaml

```diff
@@ -0,0 +1,14 @@
+---
+prelude: >
+    Compatibility of pulse parameter assignment with `ParameterVector` class.
+features:
+  - |
+    The `ParameterManager` now supports the `ParameterVector` class. This enables
+    the use of `ParameterVector` in pulse `Schedule`and `ScheduleBlock` objects when using their method
+    `assign_parameters` to assign simultaneously a list of parameter values.
+  - |
+    An additional method `_unroll_param_dict`has been added to the `ParameterManager` class to unroll the
+    parameter dictionary into a list of parameter values. This method is used to convert the parameter dictionary
+    into a list of parameter values when using the `ParameterVector` class. This method is a replication of the
+    analogous method `_unroll_param_dict` in `QuantumCircuit` class.
```

Comment:
This is good to note in the PR but doesn't need to be in the release notes since it is an internal helper function and the release notes are user facing.

```suggestion
```